website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -38,7 +38,15 @@ There are no prerequisites for running the WiFi Shield with Cyton or Ganglion on
 
 ### Linux Users: Serial Port Permissions
 
-<p /><details><summary><b><i>Attention Linux Users: Expand this dropdown for important details. This fix affects all communications between Ganglion/Cyton and your computer via serial ports.</i></b></summary><br />
+<p /><details>
```

Comment:
`<p />` is a strange html usage, but seems necessary here for the MDX interpreter. Moving it to it's own line. https://stackoverflow.com/a/1261157